frontend/public/components/utils/resource-dropdown.jsx

```diff
@@ -0,0 +1,175 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import * as fuzzy from 'fuzzysearch';
+import * as PropTypes from 'prop-types';
+import { Firehose, LoadingInline, Dropdown, ResourceName } from '.';
+
+/* Component ResourceDropdown - creates a dropdown list of resources specified */
+
+class ResourceDropdown_ extends React.Component {
```

Comment:
Was this pulled directly as-is from the binding page or were there other changes?

Reply:
Some minor tweaks but otherwise it is identical.